example-KG-COVID-19-usage.ipynb

```diff
@@ -0,0 +1,577 @@
+    "\n",
+    "##### Note: consider running the code in these notebooks on a server with GPUs in order to complete in a reasonable amount of time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use prebuilt SPARQL queries to query our Blazegraph endpoint on the commandline\n",
+    "\n",
+    "KG-COVID-19 has tooling to query our Blazegraph endpoint using predetermined SPARQL queries, and emit the results as a TSV file. Different SPARQL queries on our endpoint or other endpoints can be used by creating a new YAML file and specifying this file with the `-y` flag. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/home/reeseju/kg-covid-19/kg_covid_19/query.py:17: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\r\n",
+      "  return yaml.load(open(yaml_file))\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python run.py query -y queries/sparql/query-01-bl-cat-counts.yaml # or make a new YAML file and write your own query"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['v1', 'v0']\n",
+      "['199', 'organism taxon']\n",
+      "['19131', 'https://w3id.org/biolink/vocab/Gene']\n",
+      "['3908', 'https://w3id.org/biolink/vocab/NamedThing']\n",
+      "['20167', 'https://w3id.org/biolink/vocab/Protein']\n",
+      "['30534', 'https://w3id.org/biolink/vocab/BiologicalProcess']\n",
+      "['4468', 'https://w3id.org/biolink/vocab/CellularComponent']\n",
+      "['30018', 'https://w3id.org/biolink/vocab/ChemicalSubstance']\n",
+      "['32228', 'https://w3id.org/biolink/vocab/Drug']\n",
+      "['12241', 'https://w3id.org/biolink/vocab/MolecularActivity']\n",
+      "['62446', 'https://w3id.org/biolink/vocab/OntologyClass']\n",
+      "['6', 'https://w3id.org/biolink/vocab/OrganismalEntity']\n",
+      "['15530', 'https://w3id.org/biolink/vocab/PhenotypicFeature']\n",
+      "['129930', 'https://w3id.org/biolink/vocab/Publication']\n",
+      "['4687', 'https://w3id.org/biolink/vocab/AnatomicalEntity']\n",
+      "['48', 'https://w3id.org/biolink/vocab/Assay']\n",
+      "['703', 'https://w3id.org/biolink/vocab/Cell']\n",
+      "['24229', 'https://w3id.org/biolink/vocab/Disease']\n",
+      "['1', 'https://w3id.org/biolink/vocab/MolecularEntity']\n",
+      "['17', 'https://w3id.org/biolink/vocab/RNA']\n",
+      "['47', 'https://w3id.org/biolink/vocab/SequenceFeature']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# have a look at biolink category counts currently in KG-COVID-19 loaded on Blazegraph endpoint\n",
```

Comment:
We should link out to the biolink documentation somewhere. This notebook assumes that the user is very familiar with this, but I doubt that this will be the case for most users.

Reply:
add a bit more explanation, and a link to the Biolink model docs